airflow/utils/cli.py

```diff
@@ -238,3 +238,12 @@ def sigquit_handler(sig, frame):  # pylint: disable=unused-argument
             if line:
                 code.append("  {}".format(line.strip()))
     print("\n".join(code))
+
+
+def get_umask():
+    """
+    Returns umask to control default file permission for new files
+    """
+    cur_mask = os.umask(0)
+    os.umask(cur_mask)
```

Comment:
Why do you set it here ? I think this is not necessary.

Reply:
If I don't set it then the umask of for the rest of the execution is set to 0 and any files created by parent will be created with permissions 0666.

Reply:
I see - then it's yet another reason why we should not set/reset the umask. We really do not need to find out what is the current umask - we should set a reasonable value from configuration.

Reply:
I have removed this part. Now umask isn't read from the parent.